### ISSUE TITLE
Bluetooth toggle causing setting force closed

### ISSUE THREAD

**Issue by ghost** [Euphoria-OS/Bug-tracker#16]
Do you use Xposed framework? If yes, do not even bother reporting.
Uninstall Xposed(&reboot), check if the bug you're trying to report still
exists and then proceed. Otherwise,your report will be declared invalid.

Your device: Nexus 5

Which ROM version are you using? Latest Euphoria 03/06/2015

What kernel are you using - included in the ROM or from another source?
Provided kernel

What steps will reproduce the problem?
1. Go to settings ----then go to bluetooth (inside)
2.turn on bluetooth(inside bluetooth)---wait 1-2 for scan then turn off bluetooth (inside

)(turn on and off in bluetooth)
3. if it doesnt happend first time then try couple of times more on and off

What is the expected result? What do you see instead?
 Unfortunetly, Settings has stopped

Is the result seen after a clean flash too?
yes

Attach a logcat and/or a last_kmsg . Without this,the issue is impossible
to fix.

**Comment by GauthamYerroju** [comment]
Logcat here:
https://drive.google.com/file/d/0B8ZPEMm5hk58Zl9hSGFab2Q2TEZEUS1fZEhWSVZGVU02RUlJ/view?usp=sharing

**Comment by mitchelldavis44** [comment]
@thegwiz I'm not able to reproduce this, are you still having this issue after a clean flash with the newest build released today?

**Comment by GauthamYerroju** [comment]
@mitchelldavis44 sure, I will try a clean flash and report back in a day or two. Thanks very much for looking into this!

Also, by today's build, you mean this, right?
https://plus.google.com/117421397767192393992/posts/B52k6iK4vkG

EDIT:
Is it OK if I test it as a secondary ROM in MultiRom? Just curious if that makes any difference. If you prefer I will clean flash as a primary rom, so please do let me know.

**Comment by mitchelldavis44** [comment]
@thegwiz No, its located here https://plus.google.com/110105349583610522975/posts/KsjeTJVQMZy the one you posted was for mako. It shouldn't matter if you test in multirom or not.

**Comment by ghost** [comment]
Hi Mitch,

I have uploaded the screen recording video for FC. (latest ROM, inbulit kernel, clean flash)

Thanks for looking into it.

https://plus.google.com/103884759383887359177/posts/HJNPPBC4QNd
